src/platform/forms/save-in-progress/SaveInProgressIntro.jsx

```diff
@@ -129,11 +127,11 @@ class SaveInProgressIntro extends React.Component {
     } else if (renderSignInMessage) {
       alert = renderSignInMessage(prefillEnabled);
     } else if (prefillEnabled && !verifyRequiredPrefill) {
-      const { buttonOnly, retentionPeriod, startText } = this.props;
+      const { buttonOnly, retentionPeriod, unverifiedStartText } = this.props;
```

Comment:
If you rename this prop, please also update every usage of `<SaveInProgressIntro>` so they're not all broken.

Reply:
`startText` was added to this block a little more than a week ago since we thought it would display text when the user is not logged in....until we found out that it didn't work like that, so we created `unverifiedStartText`.